_episodes/01-intro.md

```diff
@@ -59,98 +59,61 @@ loops back and waits for you to enter another command.
 
 
 The Shell is a program which runs other programs rather than doing calculations itself.
-Those programs can be as complicated as a climate modeling software and as simple as a
-program that creates a new folder/directory. The simple programs which are used to perform
+Those programs can be as complicated as climate modeling software and as simple as a
+program that creates a new directory. The simple programs which are used to perform
 stand alone tasks are usually refered to as commands.
 The most popular Unix shell is Bash, (the Bourne Again SHell --- so-called because
 it's derived from a shell written by Stephen Bourne).
 Bash is the default shell on most modern implementations of Unix
 and in most packages that provide Unix-like tools for Windows.
 
-### What does it look like?
 
-A typical shell window looks something like:
+When the shell is first opened, you are presented with a **prompt**,
+indicating that the shell is waiting for input.
 
 ~~~
-bash-3.2$
-bash-3.2$ ls -F /
-Applications/         System/
-Library/              Users/
-Network/              Volumes/
-bash-3.2$
+$
 ~~~
+{: .language-bash}
 
-The first line shows only a **prompt**,
-indicating that the shell is waiting for input.
-Your shell may use different text for the prompt. Most importantly:
+The shell typically uses `$ ` as the prompt, but may use a different symbol.
+Most importantly:
 when typing commands, either from these lessons or from other sources,
 *do not type the prompt*, only the commands that follow it.
 
-The part that you type,
-`ls -F /` in the second line of the example,
-typically has the following structure: a **command**,
-some **options** (also called **switches** or **flags**) and an **argument**.
-Flags start with a single dash (`-`) or two dashes (`--`), and change the behaviour of a command.
-Arguments tell the command what to operate on (e.g. files and directories).
-Sometimes options and arguments are referred to as **parameters**.
-A command can be called with more than one option and more than one argument: but a
-command doesn't always require an argument or an option.
-
-In the second line of the example above, our **command** is `ls`, with an **option** `-F` and an
-**argument** `/`. Each part is separated by spaces: if you omit the space
-between `ls` and `-F` the shell will look for a command called `ls-F`, which
-doesn't exist. Also, capitalization matters: `LS` is different from `ls`.
-
-Next we see the output that our command produced. In this case it is a listing
-of files and folders in a location called `/` - we'll cover what all these mean
-later today. Those using a macOS might recognize the output in this example.
-
-Finally, the shell again prints the prompt and waits for you to type the next
-command.
-
 In the examples for this lesson, we'll show the prompt as `$ `. You can make your
```

Comment:
Minor suggestion: maybe move this into the paragraph above, immediately before the last sentence ("Most importantly:..."). Alternatively, I feel like removing this part entirely might be an improvement. It doesn't seem like there's much upside to changing their prompt to `$ `, but there is a potential big downside if they typo the command somehow and mess up their prompt (this happened to one learner the last time I taught this lesson).

Reply:
Great suggestions!